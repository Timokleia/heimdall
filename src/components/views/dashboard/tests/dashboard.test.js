import React from "react";
import Dashboard from "../dashboard";
import Enzyme, { mount, render, shallow } from "enzyme";
import Adapter from "enzyme-adapter-react-16";

Enzyme.configure({ adapter: new Adapter() });
describe("Dashboard", () => {
  it("Render without crashing", () => {
    const wrapper = shallow(<Dashboard />);
    expect(wrapper).toMatchSnapshot();
  });
});
