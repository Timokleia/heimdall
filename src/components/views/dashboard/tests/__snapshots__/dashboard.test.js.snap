// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Dashboard Render without crashing 1`] = `
<CardGroup>
  <Card
    body={false}
  >
    <CardHeader>
      Currencies
    </CardHeader>
    <DashboardCurrencies />
  </Card>
  <Card
    body={false}
  >
    <CardHeader>
      Zone accesses
    </CardHeader>
    <DashboardZoneAccesses />
  </Card>
</CardGroup>
`;
